### PR TITLE
Use the latest Cosmiframe version with security features

### PR DESCRIPTION
**Pull request by noahsaso**
## Description

Cosmiframe previously lacked origin restrictions that meant any parent window could control the app, potentially allowing for MITM attacks.

This updates it to the latest version which requires strict origin filters.

Since Graz does not currently provide any way for clients to specify custom options for a given wallet adapter, I just changed the Cosmiframe wallet to a DAO DAO-specific wallet which only works on trusted DAO DAO domains. This can be extended later if other parent apps want to integrate with Graz.

### CODE REVIEW

File: packages/graz/src/actions/wallet/daoDao.ts [hunk 1]
```diff
@@ -9,21 +9,21 @@ import type { Wallet } from "../../types/wallet";
...
-export const getCosmiframe = (): Wallet => {
-  if (window.parent === window.self) {
+export const getDaoDao = (): Wallet => {
+  if (!isInIframe()) {
     useGrazInternalStore.getState()._notFoundFn();
     throw new Error("not in iframe");
   }
 
-  const keplr = new Cosmiframe().getKeplrClient();
+  const keplr = new Cosmiframe(["https://daodao.zone", "https://dao.daodao.zone"]).getKeplrClient();
```
Comment (codingki) [inline packages/graz/src/actions/wallet/daoDao.ts]: what if we expose this options in `GrazProvider`?

Comment (noahsaso) [reply packages/graz/src/actions/wallet/daoDao.ts]: oh sure that'd be great

Comment (noahsaso) [reply packages/graz/src/actions/wallet/daoDao.ts]: done!

**Review by codingki — APPROVED** [review]
lgtm

**Pull request merged**